src/coreComponents/physicsSolvers/multiphysics/PoroelasticSolver.cpp

```diff
@@ -279,7 +279,7 @@ void PoroelasticSolver::updateDeformationForCoupling( DomainPartition & domain )
     arrayView1d< real64 > const &
     dVol = elementSubRegion.getReference< array1d< real64 > >( SinglePhaseBase::viewKeyStruct::deltaVolumeString );
 
-    arrayView1d< real64 const > const & bulkModulus = solid.getReference< array1d< real64 > >( "BulkModulus" );
+    arrayView1d< real64 const > const & bulkModulus = solid.getReference< array1d< real64 > >( "bulkModulus" );
```

Comment:
Yes, I've made it `ElasticIsotropic::viewKeyStruct::bulkModulusString`, but you can see it assumes that field will exist in other material models.  For a damage model, as well, this would give the reference bulk modulus `K0`, when what we want here is probably `(1-d)K0`.  In a follow on PR I'd like to revisit all these "external" references to constitutive properties, as it is easy to break things.